src/client/api.ts

```diff
@@ -695,20 +682,25 @@ const getLPTokenPrice = multiAsync(
 );
 
 const getUserLPTokenAmount = multiAsync(
-	async (connection: Connection, wallet: typeof Wallet, globalMarketSeed: string) => {
+	async (connection: Connection, wallet: Wallet, globalMarketSeed: string) => {
 		const lpAssociatedAddress = await getInvestorLPAssociatedTokenAddress(
 			connection,
 			wallet,
 			globalMarketSeed
 		);
 
-		const lpBalance = await connection.getTokenAccountBalance(lpAssociatedAddress);
-		return new Big(lpBalance.value.amount);
+		try {
```

Comment:
why are we adding this try catch?

Reply:
I understand what it does but I'm not sure why we are hiding errors here

Reply:
It throws an unexpected error if I don't do that, and the UI freezes.
The error is because the token account does not exist for the address